packages/components/src/slot-fill/bubbles-virtually/slot.tsx

```diff
@@ -41,7 +49,11 @@ function Slot(
 	} );
 
 	return (
-		<Component ref={ useMergeRefs( [ forwardedRef, ref ] ) } { ...props } />
+		<View
+			as={ Component }
+			ref={ useMergeRefs( [ forwardedRef, ref ] ) }
+			{ ...restProps }
+		/>
```

Comment:
When WordPressComponentProps is IsPolymorphic, using with ref,

`TS2590: Expression produces a union type that is too complex to represent.`

error occurs, so we use View as a workaround.

Reply:
Is this something we should fix separately?